### PR TITLE
Use NaN and make node-celt work with node 0.12.x

### PR DESCRIPTION
**Pull request by Prior99**
Using NaN instead of old deprecated APIs to make node-celt work with node 0.12.x.

### CODE REVIEW

File: binding.gyp [hunk 1]
```diff
@@ -40,7 +40,8 @@
             'include_dirs': [
                 '/home/wace/dev/nvm/src/node-v0.10.5/src',
                 '/home/wace/dev/nvm/src/node-v0.10.5/deps/v8/include',
-                '/home/wace/dev/nvm/src/node-v0.10.5/deps/uv/include'
+                '/home/wace/dev/nvm/src/node-v0.10.5/deps/uv/include',
+		"<!(node -e \"require('nan')\")"
```
Comment (Rantanen) [inline binding.gyp]: While the indentation seems to be a bit of a mess in the project, binding.gyp seems to be using spaces in general. The new line is indented with tabs though.

.. also I had no idea I still had my old include dirs here which probably shouldn't be there!

Any chance you could change the indent for the `require('nan')` to use spaces and while doing so remove the old `/home/wace` includes?

**Comment by Rantanen** [comment]
Thanks! Looks good save for one small nitpick. Unfortunately that's a bit of a pet peeve for me together with spaces at the end of lines. :)

If you want this merged quickly, could you tweak the bindings.gyp? If you're fine waiting, I can do it myself as well, but could take a bit.

**Comment by Prior99** [comment]
@Rantanen Done :) I didn't see this as normally my editor should use the indetionmode of the file and I had invisibles off.

**Comment by Rantanen** [comment]
For some reason the c++ files use tabs and the bindings.gyp uses spaces. No idea what the JavaScript files use.. I guess I started this project before adopting editorconfig to force proper settings. Oh well - fixing that will be for another day. :)

Thanks for the PR!

**Comment by Prior99** [comment]
Thanks for the merge. As for the indention: Happens to me all the time, some editors just detect this wrong. Main thing is that it is consistent within each file imo.

**Comment by Rantanen** [comment]
Oh. Was the package.json change from node 0.10.0 to 0.12.0 required? Didn't see such change in the jitterbuffer PR. I'm fine either way, but just curious why one of them had the change and the other one didn't. :)

**Comment by Prior99** [comment]
@Rantanen That's actually correct, jitterbuffer also needs node >= 0.12.0

**Comment by Rantanen** [comment]
Okay, will change that before updating npm. Thanks!

**Comment by Rantanen** [comment]
I just tested this with node 0.10 and it seemed to work just fine.

**Comment by Prior99** [comment]
Are you sure? I have no old node installation here, but if it works...

**Comment by Prior99** [comment]
Then please just close Rantanen/node-jitterbuffer #7

**Comment by Rantanen** [comment]
Ah - figured there was a known issue. But yeah, seems to work so I'll keep these both at >=0.10. :)

Thanks again!

**Comment by Prior99** [comment]
Perfect! I'd love to see this on npm to be abled to work with node-mumble.

**Comment by Rantanen** [comment]
Just published. Should be there now.

**Comment by Prior99** [comment]
Thank you!

**Pull request merged**